srcpkgs/wluma/template

```diff
@@ -0,0 +1,18 @@
+# Template file for 'wluma'
+pkgname=wluma
+version=1.2.2
+revision=1
+build_style=gnu-makefile
```

Comment:
Why not `build_style=meson`?

Reply:
@abenson because it doesn't work, not sure why:
```
[1/1] Linking target wluma
FAILED: wluma 
cc  -o wluma wluma.p/src_main.c.o -flto -Wl,--as-needed -Wl,--no-undefined -fstack-clash-protection -D_FORTIFY_SOURCE=2 -mtune=generic -O2 -pipe -fuse-ld=lld -Wl,-z,relro -Wl,-z,now -Wl,--as-needed -Wl,--start-group protocol/libclient_protos.a /usr/lib64/libvulkan.so -lm /usr/lib64/libwayland-client.so -Wl,--end-group -Wl,-rpath,/usr/lib64 -Wl,-rpath-link,/usr/lib64
ld.lld: error: undefined symbol: main
>>> referenced by start.S:104 (../sysdeps/x86_64/start.S:104)
>>>               /usr/lib64/gcc/x86_64-unknown-linux-gnu/10.2.1/../../../../lib64/Scrt1.o:(_start)
collect2: error: ld returned 1 exit status
ninja: build stopped: subcommand failed.
```

Reply:
I'm having trouble recreating that issue.

Reply:
```
=> wluma-1.2.2_1: running do_build ...
ninja: Entering directory `build'
[1/7] Generating 'protocol/libclient_protos.a.p/wlr-export-dmabuf-unstable-v1-protocol.c'.
[2/7] Generating 'protocol/libclient_protos.a.p/wlr-export-dmabuf-unstable-v1-client-protocol.h'.
[3/7] Generating 'wluma.p/wlr-export-dmabuf-unstable-v1-client-protocol.h'.
[4/7] Compiling C object protocol/libclient_protos.a.p/meson-generated_wlr-export-dmabuf-unstable-v1-protocol.c.o
[5/7] Linking static target protocol/libclient_protos.a
[6/7] Compiling C object wluma.p/src_main.c.o
[7/7] Linking target wluma
```

Reply:
Somehow the build fails when I use `lld` linker. I use it for my local builds because it's faster than GNU `ld`. With the default GNU `ld` `wluma` builds just fine.